### PR TITLE
Implement SIMD Optimization for Vector Classes #106

### PR DESCRIPTION
**Pull request by Patti88**
## Summary by CodeRabbit

- **New Features**
  - Introduced a resilient server communication component that automatically retries data requests when temporary busy conditions occur, ensuring a smoother experience.
  - Enhanced the retry logic with dynamic delay adjustments to better manage high-demand scenarios and improve overall reliability.
  - Added a configuration class for retry parameters, providing default settings for maximum attempts and delays.
  - Added a new exception class for better error handling during busy server conditions.

- **Bug Fixes**
  - Implemented handling for server busy conditions through a dedicated exception.

- **Tests**
  - Implemented comprehensive tests to validate the improved handling of transient busy periods and confirm successful recovery after retries.
  - Added tests for scenarios involving server busy conditions and successful retries.
  - Enhanced test coverage for the retry logic with various conditions and edge cases.

### CODE REVIEW

File: src/main/java/org/fungover/breeze/control/RetryExecutor.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
+package org.fungover.breeze.control;
+
+import java.security.SecureRandom;
+import java.util.Objects;
+import java.util.function.Function;
+
+public class RetryExecutor {
+    private static final SecureRandom JITTER = new SecureRandom();
```
Comment (Stelle83) [inline src/main/java/org/fungover/breeze/control/RetryExecutor.java:8]: Good use of SecureRandom for cryptographic randomness!

**Review by Stelle83 — DISMISSED** [review]
Consider adding more descriptive documentation/comments for better understanding of the methods and their purposes.

Overall, the changes improve code robustness, readability, and maintainability.

**Review by kappsegla — CHANGES_REQUESTED** [review]
@Patti88 It looks like there are 2 deleted files in this PR? They should probably not be deleted? Also the title of the pull request doesn't seem to have anything to do with the actual implementation?

**Review by AfagMamedova — COMMENTED** [review]
Hej!
Jag tycker det här ser bra ut och det är kul att du försöker göra koden snabbare med SIMD.
Tydlig och snygg kod, lätt att följa. Bra idé att försöka göra beräkningar snabbare.
Kanske lägga till en alternativ lösning för datorer som inte kan köra SIMD?
Några tester vore bra, så man ser att allt fungerar som det ska.
